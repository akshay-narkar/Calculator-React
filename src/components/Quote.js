export default function Quote() {
  const heading = {
    color: 'black',
    fontSize: 30,
    marginBottom: 50,
    display: 'block',
    marginTop: 150,
    paddingLeft: 50,
    paddingRight: 50,
  };

  return (
    <>
      <h4 style={heading}>
        Mathematics is not about numbers, equations, computations, or algorithms:
        it is about understanding. â€“ William Paul Thurston
      </h4>

    </>

  );
}
