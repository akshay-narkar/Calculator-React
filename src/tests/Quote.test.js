import renderer from 'react-test-renderer';
import Quote from '../components/Quote'

it('UI is unaltered wrt first snapshot', () => {

  const elem = renderer.create(<Quote/>).toJSON();

  expect(elem).toMatchSnapshot();

});