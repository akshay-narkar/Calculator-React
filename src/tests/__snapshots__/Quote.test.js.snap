// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
<h4
  style={
    Object {
      "color": "black",
      "display": "block",
      "fontSize": 30,
      "marginBottom": 50,
      "marginTop": 150,
      "paddingLeft": 200,
      "paddingRight": 200,
    }
  }
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</h4>
`;
